src/components/Thread/Thread.js

```diff
@@ -211,13 +211,13 @@ class ThreadInner extends React.PureComponent {
             Message={this.props.Message}
             {...this.props.additionalMessageListProps}
           />
-          {smartRender(this.props.MessageInput, {
-            MessageInputSmall,
-            parent: this.props.thread,
-            focus: this.props.autoFocus,
-            ...this.props.additionalMessageInputProps,
-          })}
         </div>
+        {smartRender(this.props.MessageInput, {
+          MessageInputSmall,
+          parent: this.props.thread,
+          focus: this.props.autoFocus,
+          ...this.props.additionalMessageInputProps,
+        })}
```

Comment:
whitespace change?

Reply:
The message input was moved outside of the parent div. That ways it doesn't scroll with the rest of the thread

Reply:
oh right! been staring at screen for so long